native/common/include/jp_methodoverload.h

```diff
@@ -65,7 +65,7 @@ public :
 
 	string getArgumentString();
 
-  void packArgs(JPMallocCleaner<jvalue>& v, vector<HostRef*>& arg, size_t skip);

```

Comment:
I removed MallocCleaner mostly has it is similar enough to vector that it wasn't of much use.  There was one edge case in that VC++ in older versions does no handle 0 length vectors well.  So I added an extra as a work around for now.